rocAL/source/pipeline/master_graph.cpp

```diff
@@ -437,6 +437,11 @@ MasterGraph::mem_type() {
     return _mem_type;
 }
 
+uint
```

Comment:
Move uint to L441

Reply:
every function seems to be present in this way in master_graph.cpp